code-challenges.js

```diff
@@ -2,8 +2,14 @@
 
 
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
-
-
+const fibo = (num) => {
```

Comment:
fibo function takes an argument which sets the index cap (9 = to a length of 10 numbers)